include/dev/dram/device.h

```diff
@@ -21,23 +21,47 @@
  *
  * mr2      Stores current value of MR2 in DRAM.
  * mr13     Stores current value of MR13 in DRAM.
+ * cal      Pointer to current DRAM freq cal structure.
+ * cfg      Pointer to current DRAM freq cfg structure.
  */
 typedef struct dram_dev_t
 {
-    uint8_t mr2;
-    uint8_t mr13;
+    uint8_t         mr2;
```

Comment:
shouldn't MR2 be deleted based on the comment you left below?

Reply:
Yes, MR2 should have been deleted, fixed.